payments-core-testing/api/payments-core-testing.api

```diff
@@ -0,0 +1,27 @@
+public final class com/stripe/android/testing/BuildConfig {
```

Comment:
Since we don't publish this, we should add this module to https://github.com/stripe/stripe-android/blob/7144e5cf506f94b33d9c0ce8ab11ee333b18abff/build.gradle#L145 instead.